procedures/unit-testing-utils.ipf

```diff
@@ -188,4 +188,30 @@ static Function/T GetFunctionTagValue(funcName, tagName, err)
 	err = UTF_TAG_OK
 	return tagValue
 End
+
+/// @brief Prepare the passed string for output
+///
+/// We return a fixed string if it is null and limit its size so that it can be used in a sprintf statement using plain
+/// "%s". We also do that for IP9, where this limit does not exist anymore to have a consistent output across IP
+/// versions.
+static Function/S PrepareStringForOut(str)
+	string &str
+
+	variable length
+	variable maxLength = 250
```

Comment:
How about setting this and the suffix as static constant?

Reply:
@MichaelHuth Thx. Fixed. I also explained how I reached that value.